karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       'karma-sourcemap-loader'
     ],
 
-    browsers: ['PhantomJS'],
+    browsers: process.env.CI === true ? ['PhantomJS'] : [],
```

Comment:
I feel like we're misunderstanding each other here. 

It basically boils down to this being a more sensible default. Whenever I'm developing brainstem-js (and need to debug), I have to remember to change the karma configuration file since it only specifies PhantomJS (and it is currently broken where it always does a single run). Without a browser configuration, this only sets up a testing server. The developer can then go to `localhost:9876/debug.html` to keep refreshing the tests. 

Is your opinion that `PhantomJS` as a single-run a more sensible default when developing brainstem-js?